### PR TITLE
Completed css-brochure-form exercise. - Daniel Cho

### PR DESCRIPTION
**Pull request by Daniel-J-Cho**
Screen Shot 2022-03-16 at 3 05 11 PM

### CODE REVIEW

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
Close, however there are a couple differences from the target appearance:
- The page header and submit button should be orange-red, not lime-green
- The text in the inputs, select, and submit button could use a _tiny bit_ more space above and below it.

---

image

---

image

**Comment by Daniel-J-Cho** [comment]
Screen Shot 2022-03-16 at 3 32 02 PM

I tried my best to get it to a more orange-red color.  I'm red-green color blind lol

**Comment by Daniel-J-Cho** [comment]
Screen Shot 2022-03-16 at 3 43 35 PM

Decided to re-submit because I changed the font-size property of the input, select, and submit fields in the previous submission instead of changing the height of the boxes themselves

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+h2 {
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  text-transform: uppercase;
+  color: rgb(184 144 58);
```
Comment (TimelikeClosure) [inline css-brochure-form/styles.css]: > Screen Shot 2022-03-16 at 3 32 02 PM
> 
> I tried my best to get it to a more orange-red color. I'm red-green color blind lol

In that case, there are a couple ways you can deal with issues like this.
- In a professional environment, the first thing you can try is to have the client or designer (if there is one) provide you with a color code.
- If you have to make the color more-red, you can take the `rgb` value and move some of the second number into the first number (vice versa if it needs to be more green).
- If you happen to have an in-browser example to pull from, you can use the development tools' color picker to obtain the color code from the example:

---

image

**Comment by Daniel-J-Cho** [comment]
Screen Shot 2022-03-16 at 4 06 00 PM

Ok, thank you for those tips.  Very helpful.  Is the color for this heading more accurate to orange-red?

**Comment by Daniel-J-Cho** [comment]
Screen Shot 2022-03-16 at 4 25 16 PM

**Comment by TimelikeClosure** [comment]
@DanielC2022 It's much closer (yellow-orange). The color-picker above has the hex code for it, which is equivalent to:
`rgb(236, 93, 97)`

**Comment by TimelikeClosure** [comment]
> Screen Shot 2022-03-16 at 4 25 16 PM

You just about nailed the heading in this one. Now to update the button to match.

**Comment by Daniel-J-Cho** [comment]
Screen Shot 2022-03-16 at 4 39 32 PM

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**